bigquery_etl/routine/publish_routines.py

```diff
@@ -9,17 +9,16 @@
 from google.cloud import storage
 
 from bigquery_etl.util import standard_args
-from bigquery_etl.udf.parse_udf import (
-    read_udf_dirs,
+from bigquery_etl.util.common import project_dirs
+from bigquery_etl.routine.parse_routine import (
+    read_routine_dir,
     accumulate_dependencies,
-    get_udf_dirs,
 )
 
-DEFAULT_PROJECT_ID = "moz-fx-data-shared-prod"
-DEFAULT_UDF_DIR = ["udf/", "udf_js/"]
-DEFAULT_DEPENDENCY_DIR = "udf_js/lib/"
+DEFAULT_DEPENDENCY_DIR = "lib/"
```

Comment:
A more descriptive name for this directory might be in order, since we lost the parent (`udf_js`) to tell us what the library is for.